### PR TITLE
Update Set-CsUserCallForwardingSettings.md

### PR DESCRIPTION
**Pull request by priyanshamishra**
Added a description on the -DelegateWaitRingTime parameter: The parameter DelegateWaitRingTime needs to be explicitly set to a positive value in SfB 2019 to allow call forwarding to delegates.

### CODE REVIEW

**Comment by priyanshamishra** [comment]
@MicrosoftDocs/marveldocs-pubops PR is ready for review. Please review and publish.

File: skype/skype-ps/skype/Set-CsUserCallForwardingSettings.md [hunk 1]
```diff
@@ -98,7 +98,7 @@ Accept wildcard characters: False
 ```
 
 ### -DelegateRingWaitTime
-Specifies the time, in seconds, to wait before ringing delegates. Acceptable inputs: 0, 5, 10.
+Specifies the time, in seconds, to wait before ringing delegates. Acceptable inputs: 0, 5, 10. The parameter DelegateWaitRingTime needs to be explicitly set to a positive value  to allow call forwarding to delegates.
```
Comment (get-itips) [inline skype/skype-ps/skype/Set-CsUserCallForwardingSettings.md:101]:
```suggestion
Specifies the time, in seconds, to wait before ringing delegates. Acceptable inputs: 0, 5, 10, 15. The parameter DelegateWaitRingTime needs to be explicitly set to a positive value  to allow call forwarding to delegates.
```

**Review by get-itips — CHANGES_REQUESTED** [review]
Hello @priyanshamishra how are you?
Looks like the value 15 is also accepted
image

**Review by dariomws — APPROVED** [review]
I agree with @get-itips. I'm ok after that. Thanks!

**Comment by yogkumgit** [comment]
@priyanshamishra Please update the PR with suggestions. Thanks

**Comment by priyanshamishra** [comment]
Yes, the change is alright. I have made the update from my end. Please review and publish.

**Review by get-itips — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Kateyanne Please merge this PR. Thanks

**Pull request merged**